styles/theme.scss

```diff
@@ -14,9 +14,11 @@
 
     --off-black: #0a0a0a;
     --off-white: #e8e8e8;
+    --error-font: rgb(204, 0, 0);
+    --error-font-background: rgba(255, 71, 71, 0.417);
 
+    --darkClearBg: rgba(3, 3, 3, 0.2);
```

Comment:
Not to be a total pain, but yea...let's rename this one to match the format of the other variables, using hyphens instead of camelCase. `--dark-clear-bg`